raiden_contracts/deploy/__main__.py

```diff
@@ -317,85 +339,61 @@ def verify(ctx, rpc_provider, contracts_version):
+        'block_number': endpoint_registry_receipt['blockNumber'],
+        'gas_cost': endpoint_registry_receipt['gasUsed'],
+        'constructor_arguments': [],
+    }
 
     secret_registry_receipt = deployer.deploy(CONTRACT_SECRET_REGISTRY)
-    deployed_contracts[CONTRACT_SECRET_REGISTRY] = to_checksum_address(
-        secret_registry_receipt['contractAddress'],
-    )
+    deployed_contracts['contracts'][CONTRACT_SECRET_REGISTRY] = {
+        'address': to_checksum_address(
+            secret_registry_receipt['contractAddress'],
+        ),
+        'transaction_hash': encode_hex(secret_registry_receipt['transactionHash']),
+        'block_number': secret_registry_receipt['blockNumber'],
+        'gas_cost': secret_registry_receipt['gasUsed'],
+        'constructor_arguments': [],
+    }
 
     token_network_constructor_arguments = [
-        deployed_contracts[CONTRACT_SECRET_REGISTRY],
-        chain_id,
+        deployed_contracts['contracts'][CONTRACT_SECRET_REGISTRY]['address'],
+        deployed_contracts['chain_id'],
         DEPLOY_SETTLE_TIMEOUT_MIN,
         DEPLOY_SETTLE_TIMEOUT_MAX,
     ]
     token_network_registry_receipt = deployer.deploy(
         CONTRACT_TOKEN_NETWORK_REGISTRY,
         token_network_constructor_arguments,
     )
-    deployed_contracts[CONTRACT_TOKEN_NETWORK_REGISTRY] = to_checksum_address(
-        token_network_registry_receipt['contractAddress'],
-    )
-
-    deployed_contracts_info = {
-        'contracts_version': deployer.contract_manager.contracts_version,
-        'chain_id': chain_id,
-        'contracts': {
-            CONTRACT_ENDPOINT_REGISTRY: {
-                'address': deployed_contracts[CONTRACT_ENDPOINT_REGISTRY],
-                'transaction_hash': encode_hex(endpoint_registry_receipt['transactionHash']),
-                'block_number': endpoint_registry_receipt['blockNumber'],
-                'gas_cost': endpoint_registry_receipt['gasUsed'],
-                'constructor_arguments': [],
-            },
-            CONTRACT_SECRET_REGISTRY: {
-                'address': deployed_contracts[CONTRACT_SECRET_REGISTRY],
-                'transaction_hash': encode_hex(secret_registry_receipt['transactionHash']),
-                'block_number': secret_registry_receipt['blockNumber'],
-                'gas_cost': secret_registry_receipt['gasUsed'],
-                'constructor_arguments': [],
-            },
-            CONTRACT_TOKEN_NETWORK_REGISTRY: {
-                'address': deployed_contracts[CONTRACT_TOKEN_NETWORK_REGISTRY],
-                'transaction_hash': encode_hex(token_network_registry_receipt['transactionHash']),
-                'block_number': token_network_registry_receipt['blockNumber'],
-                'gas_cost': token_network_registry_receipt['gasUsed'],
-                'constructor_arguments': token_network_constructor_arguments,
-            },
-        },
+    deployed_contracts['contracts'][CONTRACT_TOKEN_NETWORK_REGISTRY] = {
```

Comment:
The deployment data dict format remains the same here, right?

Reply:
Yes. 
I only changed how it is built, after moving `store_deployment_info` and `verify_deployed_contracts` out off this deployment function.